src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -24,14 +25,17 @@ export const ConversationSpace: React.FunctionComponent = () => {
   const venue = useSelector(currentVenueSelectorData);
   const { recentVenueUsers } = useRecentVenueUsers();
 
+  const defaultTables = venue?.config?.tables ?? TABLES;
+  const [tables, setTables] = useState(defaultTables);
+
+  useEffect(() => setTables(defaultTables), [defaultTables]);
```

Comment:
There is duplicated logic between this component and JazzTab. They both have `useState` `defaultTables` and `useEffect` instead of accepting `defaultTables` is it possible to make it accept `venue.config.tables` and move this whole logic into the `TablesControlBar` That way we won't have to do this twice.

Reply:
@yurii-lubynets ^ Not really a change request, more of a comment.

Reply:
Actually,  `tables` and `jazzbarTables` are used for JazzTab and ConversationSpace components, that's why extracting them into `TablesControlBar` could be redundant